doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst

```diff
@@ -327,3 +327,47 @@ The CSIP implementation supports the following roles
 
 The API reference for media control can be found in
 :ref:`Bluetooth Coordinated Sets <bluetooth_coordinated_sets>`.
+
+
+Things worth knowing or considering when using LE Audio
+=======================================================
+
+This section describes a few tings to consider when contributing to or using LE Audio in Zephyr.
+The things described by this section is not unique to Zephyr as they are defined by the
+specifications.
+
+Security requirements
+---------------------
+
+All LE Audio services require Security Level 2 but where the key must be 128-bit and derived via an
+OOB method or via LE Secure connection.
+There is no Core-spec defined way of reporting this or otherwise achieving it,
```

Comment:
These 4 lines are a bit confusing. You mean to say that there is no core-spec defined way of reporting the 128 bit key?
Also,  'Since the security requirements' refers to the 128 bit key?

Reply:
Good point. Rewrote it a bit